src/VisualStudio/Core/Def/Implementation/SymbolSearch/RoslynSymbolResult.cs

```diff
@@ -120,7 +152,13 @@ internal static async Task<RoslynSymbolResult> MakeAsync(RoslynSymbolSource symb
 
         public ImageId DefinitionIcon { get; private set; }
 
-        public string ProjectName { get; set; }
+        public string ProjectName { get; private set; }
+
+        public string Kind { get; private set; }
+
+        public string ContainingTypeName { get; private set; }
+
+        public string ContainingMemberName { get; private set; }
```

Comment:
how do these members get pulled in by VS?  are they just hardcoded?  That seems highly problematic.  DIfferent languages may not have these concepts.  or it may have deeper concepts (like there may be no containing types, and many containing members).